testing/correctness/apps/sequence_window_python/sequence_window.py

```diff
@@ -27,7 +27,9 @@ def application_setup(args):
     ab = wallaroo.ApplicationBuilder("Sequence Window")
     ab.new_pipeline("Sequence Window",
                     wallaroo.TCPSourceConfig(in_host, in_port, decoder))
+    ab.to_stateful(count_values, Counter, "Counter")
     ab.to(maybe_one_to_many)
+    ab.to_parallel(id)
```

Comment:
This is going to break the consistency check in multi-worker runs.
I think you should either use a different app, or write a new one that doesn't have topology-sensitive validation (like the order-of-arrival sequence validation in this one)